src/sys/socket/sockopt.rs

```diff
@@ -905,18 +905,31 @@ sockopt_impl!(
     libc::IP_PKTINFO,
     bool
 );
-#[cfg(any(linux_android, target_os = "freebsd", apple_targets, netbsdlike))]
+#[cfg(any(linux_android))]
```

Comment:
I removed BSD target because I can't find `IPV6_RECVPKTINFO` in `ip6` bsd man pages. https://man.freebsd.org/cgi/man.cgi?query=ip6&sektion=4

Reply:
It is available on FreeBSD: https://github.com/freebsd/freebsd-src/blob/ef4f4a44d9134ddadede0e2e6e658d0688c5ab3c/sys/netinet6/in6.h#L446

Reply:
Also defined in the libc crate: https://docs.rs/libc/latest/x86_64-unknown-freebsd/libc/constant.IPV6_RECVPKTINFO.html

Reply:
The CI failure on BSDs is introduced by this change, please revert it

Reply:
Thank you for the links! I have updated it